packages/backend/src/paymentPointerKey/service.ts

```diff
@@ -0,0 +1,88 @@
+import { TransactionOrKnex } from 'objection'
+
+import { PaymentPointerKey } from './model'
+import { BaseService } from '../shared/baseService'
+import { JWKWithRequired } from 'auth'
+
+export interface PaymentPointerKeyService {
+  create(options: CreateOptions): Promise<PaymentPointerKey>
+  revokeKeyById(keyId: string): Promise<string>
```

Comment:
I realize this is from `ClientKeysService`, but I'd prefer
```suggestion
  revoke(id: string): Promise<PaymentPointerKey | undefined>
```
especially now that `id` !== `jwk.keyid`.
It should return `undefined` if the key doesn't exist. (This needs to be tested)
I'm not sure what should happen if the key is already revoked.

Reply:
Why don't we throw when the key doesn't exist? Isn't it dangerous if the undefined value is not handled? I think we can return revoked `PaymentPointerKey`if the key is already revoked if there is not an exceptional case.